spec/abstractops.html

```diff
@@ -1146,6 +1146,11 @@ <h1>ISO 8601 grammar</h1>
       TemporalRelativeToString :
           TemporalDateTimeString
     </emu-grammar>
+    <ul>
+      <li>
+        It is a Syntax Error if |DateExtendedYear| is *"-000000"*.
```

Comment:
My other comment reminded me that **"−000000"** (U+2212 MINUS SIGN followed by 000000) should also be a Syntax Error.